example/lib/main.dart

```diff
@@ -21,6 +22,28 @@ Future<void> main() async {
   runApp(const MyApp());
 }
 
+/// This function runs in a separate isolate with its own sandboxed thread, meant to demonstate capturing exceptions from background "threads"
+void _backgroundIsolateEntryPoint(SendPort sendPort) async {
+  await Future.delayed(const Duration(milliseconds: 500));
+
+  try {
+    // Simulate an exception in the background isolate
+    throw StateError('Background isolate processing failed!');
+  } catch (e, stack) {
+    // Send exception data back to main isolate for capture
+    sendPort.send({
+      'error': e.toString(),
+      'errorType': e.runtimeType.toString(),
+      'stackTrace': stack.toString(),
+      'properties': {
+        'test_type': 'background_isolate_exception',
+        'isolate_name': 'exception-demo-worker',
+        'button_pressed': 'capture_exception_background',
+      },
+    });
+  }
+}
```

Comment:
i still find this example odd, this is not something an user would do.
Exceptions will bubble up to the main isolate anyway
whats your intention here?